tests/copilot_helper/test_command_bootstrap.py

```diff
@@ -115,6 +115,8 @@ def test_make_config(tmp_path):
     with open(str(tmp_path / "copilot/test-service/manifest.yml")) as service:
         assert service.read() == test_service_manifest
 
+    assert os.path.exists(str(tmp_path / "copilot/test-service/addons"))
```

Comment:
This test didn't make any assertions about the addons or overrides folders being created